src/pkg/egress/syslog/https_batch.go

```diff
@@ -15,6 +15,7 @@ const BATCHSIZE = 256 * 1024
 
 type HTTPSBatchWriter struct {
 	HTTPSWriter
+	*Retryer
```

Comment:
Why embed `*Retryer` rather than using a named field to encapsulate the new struct via composition?

Reply:
I was using a Pointer here, so that I could insert the retryer by creating it in the writer factory layer, so that the implementation and logic does not rely on the writers itself to propagate the settings through to the retryer.
Might be a shortcoming due to my limited understanding of Go idiomatic concepts.